src/libtomahawk/audio/AudioEngine.cpp

```diff
@@ -401,10 +401,6 @@ AudioEngine::canSeek()
 {
     Q_D( AudioEngine );
 
-    if ( !d->audioOutput->isSeekable() ) {
```

Comment:
With your current implementation isSeekable always returned true anyway because 
"Streams often can be seeked with imem even if reported as non seekable by vlc"